ValueError: dictionary doesn't specify a version

The CI pipeline is breaking with the following error. The recent change to our logging setup in PR #131 has introduced this error.

```
Run poetry run python project/manage.py test
Traceback (most recent call last):
  File "/home/runner/work/companionship-care/companionship-care/project/manage.py", line 22, in <module>
    main()
  File "/home/runner/work/companionship-care/companionship-care/project/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/runner/work/companionship-care/companionship-care/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/home/runner/work/companionship-care/companionship-care/.venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 420, in execute
    django.setup()
  File "/home/runner/work/companionship-care/companionship-care/.venv/lib/python3.9/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/runner/work/companionship-care/companionship-care/.venv/lib/python3.9/site-packages/django/utils/log.py", line [7](https://github.com/CompanionshipCare/companionship-care/actions/runs/3206268749/jobs/5239813889#step:9:8)6, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/logging/config.py", line [8](https://github.com/CompanionshipCare/companionship-care/actions/runs/3206268749/jobs/5239813889#step:9:9)0[9](https://github.com/CompanionshipCare/companionship-care/actions/runs/3206268749/jobs/5239813889#step:9:10), in dictConfig
    dictConfigClass(config).configure()
  File "/opt/hostedtoolcache/Python/3.9.[14](https://github.com/CompanionshipCare/companionship-care/actions/runs/3206268749/jobs/5239813889#step:9:15)/x64/lib/python3.9/logging/config.py", line 497, in configure
    raise ValueError("dictionary doesn't specify a version")
ValueError: dictionary doesn't specify a version
Error: Process completed with exit code 1.
```

--- a/project/core/settings.py
+++ b/project/core/settings.py
@@ -215,6 +215,7 @@ DEBUG_TOOLBAR_PANELS = [
 
 # Rich terminal output configuration
 LOGGING = {
+    "version": 1,
     "formatters": {
         "rich": {"datefmt": "[%X]"},
     },

